### ISSUE TITLE
Add CI/CD for deploying the Prow Monitoring stack

### ISSUE THREAD

**Issue by maxgio92** [falcosecurity/test-infra#1042]
**Motivation**

All setup should be automated without human intervention and public.

**Feature**

Add a CI and CD pipeline to deploy the Prow Monitoring stack.

**Alternatives**

GitOps way with a Kubernetes operator which pulls from Git repository and applies from the cluster (the pull-way).

**Additional context**

NA

**Comment by FedeDP** [comment]
I agree with going with the easiest way possible for now adding the `make` in the `tools/deploy.sh` script!
